rosette-scala/src/main/scala/coop/rchain/rosette/Misc.scala

```diff
@@ -0,0 +1,8 @@
+package coop.rchain.rosette
+
+object Misc {
+  def suicide(msg: String): Unit = {
```

Comment:
Let's put that in the `coop.rchain.rosette` package object? http://www.scala-lang.org/docu/files/packageobjects/packageobjects.html

Reply:
Done